examples/send-plugin-message/pluginMessageApp.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Html Messages</title>
+</head>
+
+<body onload="main()">
+</body>
+
+<script>
+
+  var bsMessage = new BSMessagePort();
+
+  function main() {
+    console.log("=== Starting javascript...");
+
+    bsMessage.onbsmessage = function (msg) {
+      if (msg.data["serialNumber"] !== undefined && msg.data["filename"] !== undefined) {
+        console.log("=== Received plugin message from: " + msg.data["serialNumber"]);
```

Comment:
It'd make more sense for the log message to indicate the message sent, rather than the serial number the plugin message was received from. 

The plugin message is sent and received all within the scope of the same serial number.